application/values-test.yaml

```diff
@@ -244,7 +244,10 @@ ingress:
   hosts:
     - chart-example.local
     - '{{ .Release.Name }}.stakater.com'
-  
+
+  # Additional labels for this Ingress
+  additionalLabels:
+
```

Comment:
please add some labels here for testing, so this field does not remain empty and get validated

Reply:
@usamaahmadkhan ^^